pkg/client/results/testdata/mockResults/plugins/ds-junit-02/ds-junit-02.golden.json

```diff
@@ -43,14 +49,17 @@
 },
 "items": [
 {
+"name": "testsuite-001",
+"status": "failed",
+"items": [
+{
 "name": "[k8s.io] Pods should be submitted and removed [NodeConformance] [Conformance]",
 "status": "passed"
 },
 {
 "name": "[sig-apps] Daemon set [Serial] should rollback without unnecessary restarts [Conformance]",
 "status": "failed",
 "details": {
-"failure": "/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/framework.go:696\nConformance test suite needs a cluster with at least 2 nodes.\nExpected\n    \u003cint\u003e: 1\nto be \u003e\n    \u003cint\u003e: 1\n/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/apps/daemon_set.go:385",
```

Comment:
This should still be here; not sure why it got removed. I see a few other tests seem to show the failure stuck around... Should be an easy fix when I find it.

Reply:
The problem seems to be that I was looking for the `failure.Message` but that is 0, this is the `chardata` from the xml which (in our datatype) is called `Contents`.

The fix is fast (use the new field) but raises the question about what we should do with this new field. Should we just concat the message/contents or should we save them separately? Going to take a peek at some of the other junit (non-e2e) and see what makes sense there to compare.

Reply:
:) So it seems they just use the opposite on the other output I've tested against (chef/inspec with junit output). They only use the message for the failure message and https://github.com/inspec/inspec/blob/a35d37043e65ce5fd528d58cd48ecb369c74a432/lib/inspec/reporters/junit.rb#L48-L54 seems to show they never even use the contents.

Reply:
Settled on joining message/contents together (space separated) so the data is all present. Users could still grep the data and have one main location to look still.  I thought this was better than adding a new `Details` entry for every type of attribute/field. Open to discussion though.